lighthouse-core/computed/trace-of-tab.js

```diff
@@ -19,6 +19,15 @@ class LHTraceProcessor extends TraceProcessor {
     return new LHError(LHError.errors.NO_NAVSTART);
   }
 
+  /**
+   * This isn't ever actually used, but might one day be.
```

Comment:
lol if this is true (and there aren't concrete FR plans for it), why are we refactoring _and_ adding `'firstResourceSendRequest'`? Why not refactor but have `'lastNavigationStart'` be the only option right now?

Reply:
well, looking at the excellent comment on `computeTimeOrigin`, maybe the simpler thing is to change this to something more concrete like
```suggestion
   * This isn't currently used, but will be when the time origin of trace processing is changed.
   * @see {TraceProcessor.computeTimeOrigin}
```

(the `@see` doesn't actually work like for that yet, but [will soon(!)](https://devblogs.microsoft.com/typescript/announcing-typescript-4-1-beta/#jsdoc-see-tag), though if the thing isn't imported, I don't know if you actually have to do something like `@see {import('../../path/to/trace-processor.js').computeTimeOrigin}` or what)

Reply:
> why are we [...] adding 'firstResourceSendRequest'? 

firstResourceSendRequest won't be used for FR, but will be used for the #8984 line of work that will continue amidst FR.

> Why not refactor but have 'lastNavigationStart' be the only option right now?

That's another option but a motivating example to validate the split seemed worthwhile to pair with the refactor given its simplicity and test uninterestingness if there's only one option :)

> maybe the simpler thing is to change this to something more concrete like

sg 👍